beacon_node/network/src/network_beacon_processor/gossip_methods.rs

```diff
@@ -214,6 +214,19 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
         reprocess_tx: Option<mpsc::Sender<ReprocessQueueMessage>>,
         seen_timestamp: Duration,
     ) {
+        let attestation_size = attestation.as_ssz_bytes().len() as u64;
+        let client = self.network_globals.client(&peer_id).kind.to_string();
+
+        metrics::inc_counter_vec_by(
+            &metrics::BYTES_RECEIVED_PER_CLIENT,
+            &[&client],
+            attestation_size,
+        );
+
+        metrics::inc_counter_vec(
+            &metrics::MESSAGES_RECEIVED_PER_CLIENT,
+            &[&client]
```

Comment:
You should label this metric by [client, object_type] where object type could be
- `gossip_attestation`
- `gossip_block`
- `range_sync_block`
- `backfill_sync_block`

etc